hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -2440,6 +2440,12 @@ public ClearDeadServersResponse clearDeadServers(RpcController controller,
         Set<Address> clearedServers = new HashSet<>();
         for (HBaseProtos.ServerName pbServer : request.getServerNameList()) {
           ServerName server = ProtobufUtil.toServerName(pbServer);
+          final boolean deadInProcess = master.getServerManager().isDeadServersInProgress(server);
```

Comment:
Why this change?

Reply:
see 

`  public synchronized boolean removeDeadServer(final ServerName deadServerName) {	  public synchronized boolean removeDeadServer(final ServerName deadServerName) {
    Preconditions.checkState(!processingServers.contains(deadServerName),	
      "Asked to remove server still in processingServers set " + deadServerName +	
          " (numProcessing=" + processingServers.size() + ")");	
    return this.deadServers.remove(deadServerName) != null;	    return this.deadServers.remove(deadServerName) != null;
  }`
 We need to check where region server with special server name is in processing, so i add such a method